tests/queries/0_stateless/02346_text_index_direct_read_array.sql

```diff
@@ -0,0 +1,89 @@
+-- Tags: no-parallel, no-parallel-replicas
+-- Tag no-parallel -- due to access to the system.text_log
+-- Tag no-parallel-replicas -- direct read is not compatible with parallel replicas
```

Comment:
As per https://github.com/ClickHouse/ClickHouse/pull/88262, this is no longer true.

@Ergus Would you remove tag `no-paralle-replicas` from this test and the other SQL tests for direct read in a follow-up PR and see what happens?